mobius/src/orthogonal_arcs.rs

```diff
@@ -1,13 +1,24 @@
 use std::f64::consts::{PI, TAU};
 
+use thiserror::Error;
+
 use crate::{
     geometry::{
-        ArcAngles, Circle, CircularArc, DirectedEdge, GeneralizedCircle, Line, LineSegment,
+        ArcAngles, ArcAnglesError, ArcDirection, Circle, CircularArc, DirectedEdge,
+        GeneralizedCircle, Line, LineSegment,
     },
     nearly::is_nearly,
     Complex,
 };
 
+#[derive(Debug, Error)]
+pub enum OrthogonalArcError {
+    #[error("{0}")]
+    BadAngles(#[from] ArcAnglesError),
+    #[error("a and b must be distinct: {0}")]
+    DuplicateInt(i64),
+}
+
 pub fn compute_orthogonal_circle(
```

Comment:
Also make circular versions of the integer arcs. The integers can be mapped to the unit circle by computing $\omega^k$ where $\omega$ is an `n-th` root of unity

Reply:
🤔 at that point... maybe it would be better to make a separate file for integer arcs.